Tools/i18n/.ruff.toml

```
@@ -0,0 +1,9 @@
+extend = "../../.ruff.toml"  # Inherit the project-wide settings
+target-version = "py313"
+
+[lint]
+select = [
+    "F",  # Enable all pyflakes rules
+    "I",  # Enable all isort rules
+    "UP",  # Enable all pyupgrade rules by default
+]
```

consistency with other `.ruff.toml`
```suggestion
[lint]
select = [
    "F",     # pyflakes
    "I",     # isort
    "UP",    # pyupgrade
]
```